From 78a2ea69e0f7cdbdf838570181d33bca4a6d4d45 Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Wed, 17 Sep 2014 16:24:08 -0400
Subject: [redhat] kabi: Adding symbol module_refcount (kernel/module.c)

Message-id: <37d5b2948974570006dbf45457c42b6a95ad27dd.1410510773.git.poros@redhat.com>
Patchwork-id: 95992
O-Subject: [PATCH RHEL7.1 128/233] kabi: Adding symbol module_refcount (kernel/module.c)
Bugzilla: 1071098
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

Bugzilla: 1071098
    https://bugzilla.redhat.com/show_bug.cgi?id=1071098

    Description:
    ============
    Used to get the refcount of the fencing module when the module is
    configured and unconfigured (used to log to the vxfen debug log)
    this is required for debugging purposes if the module fails to unload.

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/module_refcount
 create mode 100644 redhat/kabi/kabi-module/kabi_s390x/module_refcount
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/module_refcount

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/module_refcount b/redhat/kabi/kabi-module/kabi_ppc64/module_refcount
new file mode 100644
index 0000000..b45d9d3
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/module_refcount
@@ -0,0 +1,2 @@
+#1-
+0x89b3f6dc module_refcount vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_s390x/module_refcount b/redhat/kabi/kabi-module/kabi_s390x/module_refcount
new file mode 100644
index 0000000..60d4804
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/module_refcount
@@ -0,0 +1,2 @@
+#1-
+0xa93dab8a module_refcount vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/module_refcount b/redhat/kabi/kabi-module/kabi_x86_64/module_refcount
new file mode 100644
index 0000000..3bf651f
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/module_refcount
@@ -0,0 +1,2 @@
+#1-
+0xd3d14f23 module_refcount vmlinux EXPORT_SYMBOL
-- 
1.7.1